tests/functional/pv/pv_encryption/test_encrypted_pvc_no_expansion_option.py

```diff
@@ -0,0 +1,144 @@
+import logging
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import green_squad
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier1,
+    skipif_ocs_version,
+    kms_config_required,
+    skipif_managed_service,
+    skipif_hci_provider_and_client,
+    skipif_disconnected_cluster,
+    skipif_proxy_cluster,
+    config,
+)
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import CommandFailed
+
+
+log = logging.getLogger(__name__)
+
+# Set the arg values based on KMS provider.
+if config.ENV_DATA["KMS_PROVIDER"].lower() == constants.HPCS_KMS_PROVIDER:
+    kmsprovider = constants.HPCS_KMS_PROVIDER
+    argnames = ["kv_version", "kms_provider"]
+    argvalues = [
+        pytest.param("v1", kmsprovider),
+    ]
+else:
+    kmsprovider = constants.VAULT_KMS_PROVIDER
```

Comment:
Should have a condition for if KMS provider doesn't exist

Reply:
I added a marker for tests  @kms_config_required,  so it will  skip tests if KMS config is not defined.